### PR TITLE
Move mock control msg to API

### PR DESCRIPTION
**Pull request by Manishearth**
Otherwise script needs to depend on the backends

### CODE REVIEW

File: rust-webvr-api/src/lib.rs [hunk 1]
```diff
@@ -23,6 +23,8 @@ extern crate ipc_channel;
 
 extern crate gleam;
 
+
+pub mod mock;
```
Comment (jdm) [inline rust-webvr-api/src/lib.rs]: Looks like this file is missing.

**Comment by Manishearth** [comment]
fixed
-Manish Goregaokar

File: rust-webvr-api/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rust-webvr-api"
-version = "0.11.0"
+version = "0.11.3"
```
Comment (Eijebong) [inline rust-webvr-api/Cargo.toml]: Why not 0.11.1 ?

**Comment by asajeffrey** [comment]
LGTM, you can r=me once the version number is sorted.

File: rust-webvr-api/Cargo.toml (hunk 2 above)
Comment (Manishearth) [reply rust-webvr-api/Cargo.toml]: Keeping the two numbers in sync

**Comment by Manishearth** [comment]
@bors r=asajeffrey

changed it to 11.1

**Pull request merged**